diff -urN qt-everywhere-opensource-src-5.3.1/qttools/src/linguist/Qt5LinguistToolsConfig.cmake.in qt-everywhere-opensource-src-5.3.1-patched/qttools/src/linguist/Qt5LinguistToolsConfig.cmake.in
--- qt-everywhere-opensource-src-5.3.1/qttools/src/linguist/Qt5LinguistToolsConfig.cmake.in	2014-06-19 21:08:27.000000000 +1100
+++ qt-everywhere-opensource-src-5.3.1-patched/qttools/src/linguist/Qt5LinguistToolsConfig.cmake.in	2014-06-26 15:57:26.166772101 +1100
@@ -15,7 +15,7 @@
 if(_realCurr STREQUAL _realOrig)
     get_filename_component(_qt5_linguisttools_install_prefix \"$$CMAKE_INSTALL_LIBS_DIR/$${CMAKE_RELATIVE_INSTALL_LIBS_DIR}\" ABSOLUTE)
 else()
-    get_filename_component(_qt5_linguisttools_install_prefix \"${CMAKE_CURRENT_LIST_DIR}/$${CMAKE_RELATIVE_INSTALL_DIR}\" ABSOLUTE)
+    get_filename_component(_qt5_linguisttools_install_prefix \"${CMAKE_CURRENT_LIST_DIR}/$${CMAKE_RELATIVE_INSTALL_DIR}../../qt5\" ABSOLUTE)
 endif()
 unset(_realOrig)
 unset(_realCurr)
@@ -45,7 +45,7 @@
     add_executable(Qt5::lrelease IMPORTED)
 
 !!IF isEmpty(CMAKE_BIN_DIR_IS_ABSOLUTE)
-    set(imported_location \"${_qt5_linguisttools_install_prefix}/$${CMAKE_BIN_DIR}lrelease$$CMAKE_BIN_SUFFIX\")
+    set(imported_location \"${_qt5_linguisttools_install_prefix}/bin/lrelease$$CMAKE_BIN_SUFFIX\")
 !!ELSE
     set(imported_location \"$${CMAKE_BIN_DIR}lrelease$$CMAKE_BIN_SUFFIX\")
 !!ENDIF
@@ -60,7 +60,7 @@
     add_executable(Qt5::lupdate IMPORTED)
 
 !!IF isEmpty(CMAKE_BIN_DIR_IS_ABSOLUTE)
-    set(imported_location \"${_qt5_linguisttools_install_prefix}/$${CMAKE_BIN_DIR}lupdate$$CMAKE_BIN_SUFFIX\")
+    set(imported_location \"${_qt5_linguisttools_install_prefix}/bin/lupdate$$CMAKE_BIN_SUFFIX\")
 !!ELSE
     set(imported_location \"$${CMAKE_BIN_DIR}lupdate$$CMAKE_BIN_SUFFIX\")
 !!ENDIF
@@ -75,7 +75,7 @@
     add_executable(Qt5::lconvert IMPORTED)
 
 !!IF isEmpty(CMAKE_BIN_DIR_IS_ABSOLUTE)
-    set(imported_location \"${_qt5_linguisttools_install_prefix}/$${CMAKE_BIN_DIR}lconvert$$CMAKE_BIN_SUFFIX\")
+    set(imported_location \"${_qt5_linguisttools_install_prefix}/bin/lconvert$$CMAKE_BIN_SUFFIX\")
 !!ELSE
     set(imported_location \"$${CMAKE_BIN_DIR}lconvert$$CMAKE_BIN_SUFFIX\")
 !!ENDIF
