diff -urN qt-everywhere-opensource-src-5.1.0/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in qt-everywhere-opensource-src-5.1.0-patched/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in
--- qt-everywhere-opensource-src-5.1.0/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in	2013-07-02 18:09:32.000000000 +1100
+++ qt-everywhere-opensource-src-5.1.0-patched/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in	2013-07-19 00:58:16.535881869 +1100
@@ -49,7 +49,7 @@
 !!IF isEmpty(CMAKE_DLL_DIR_IS_ABSOLUTE)
     set(imported_location \"${_qt5$${CMAKE_MODULE_NAME}_install_prefix}/$${CMAKE_DLL_DIR}${LIB_LOCATION}\")
 !!ELSE
-    set(imported_location \"IMPORTED_LOCATION_${Configuration}\" \"$${CMAKE_DLL_DIR}${LIB_LOCATION}\")
+    set(imported_location \"$${CMAKE_DLL_DIR}${LIB_LOCATION}\")
 !!ENDIF
     _qt5_$${CMAKE_MODULE_NAME}_check_file_exists(${imported_location})
     set_target_properties(Qt5::$${CMAKE_MODULE_NAME} PROPERTIES
