diff --git a/qtwayland/examples/wayland/wayland.pro b/qtwayland/examples/wayland/wayland.pro
index d633fb9..5bdc471 100644
--- a/qtwayland/examples/wayland/wayland.pro
+++ b/qtwayland/examples/wayland/wayland.pro
@@ -1,13 +1,9 @@
 TEMPLATE=subdirs
 
-#Only build compositor examples if we are building
-#the QtCompositor API
-contains(CONFIG, wayland-compositor) {
-    SUBDIRS += qwindow-compositor
+SUBDIRS += qwindow-compositor
 
-    qtHaveModule(quick) {
-        SUBDIRS += qml-compositor
-    }
-
-    SUBDIRS += server-buffer
+qtHaveModule(quick) {
+    SUBDIRS += qml-compositor
 }
+
+SUBDIRS += server-buffer
diff --git a/qtwayland/src/plugins/hardwareintegration/hardwareintegration.pro b/qtwayland/src/plugins/hardwareintegration/hardwareintegration.pro
index 4f5ec14..59d867f 100644
--- a/qtwayland/src/plugins/hardwareintegration/hardwareintegration.pro
+++ b/qtwayland/src/plugins/hardwareintegration/hardwareintegration.pro
@@ -1,7 +1,4 @@
 TEMPLATE=subdirs
 
 SUBDIRS += client
-#The compositor plugins are only useful with QtCompositor
-contains(CONFIG, wayland-compositor) {
-        SUBDIRS += compositor
-}
+SUBDIRS += compositor
diff --git a/qtwayland/src/plugins/platforms/platforms.pro b/qtwayland/src/plugins/platforms/platforms.pro
index 0665511..28079ef 100644
--- a/qtwayland/src/plugins/platforms/platforms.pro
+++ b/qtwayland/src/plugins/platforms/platforms.pro
@@ -7,8 +7,6 @@ config_wayland_egl {
     SUBDIRS += qwayland-egl
 }
 
-#The following integrations are only useful with QtCompositor
-contains(CONFIG, wayland-compositor) {
     config_brcm_egl: \
         SUBDIRS += qwayland-brcm-egl
 
@@ -18,5 +16,4 @@ contains(CONFIG, wayland-compositor) {
         !contains(QT_CONFIG, opengles2):config_glx: \
             SUBDIRS += qwayland-xcomposite-glx
     }
-}
 
diff --git a/qtwayland/src/src.pro b/qtwayland/src/src.pro
index 332d0df..40cddf0 100644
--- a/qtwayland/src/src.pro
+++ b/qtwayland/src/src.pro
@@ -4,13 +4,10 @@ sub_qtwaylandscanner.subdir = qtwaylandscanner
 sub_qtwaylandscanner.target = sub-qtwaylandscanner
 SUBDIRS += sub_qtwaylandscanner
 
-# Don't build QtCompositor API unless explicitly enabled
-contains(CONFIG, wayland-compositor) {
-    sub_compositor.subdir = compositor
-    sub_compositor.depends = sub-qtwaylandscanner
-    sub_compositor.target = sub-compositor
-    SUBDIRS += sub_compositor
-}
+sub_compositor.subdir = compositor
+sub_compositor.depends = sub-qtwaylandscanner
+sub_compositor.target = sub-compositor
+SUBDIRS += sub_compositor
 
 sub_client.subdir = client
 sub_client.depends = sub-qtwaylandscanner
@@ -19,6 +16,6 @@ SUBDIRS += sub_client
 
 sub_plugins.subdir = plugins
 sub_plugins.depends = sub-qtwaylandscanner sub-client
-contains(CONFIG, wayland-compositor): sub_plugins.depends += sub-compositor
+sub_plugins.depends += sub-compositor
 sub_plugins.target = sub-plugins
 SUBDIRS += sub_plugins
diff --git a/qtwayland/tests/auto/auto.pro b/qtwayland/tests/auto/auto.pro
index 822fdeb..20ab95c 100644
--- a/qtwayland/tests/auto/auto.pro
+++ b/qtwayland/tests/auto/auto.pro
@@ -1,9 +1,5 @@
 TEMPLATE=subdirs
 
-#Only build compositor test when we are
-#building QtCompositor
-contains(CONFIG, wayland-compositor) {
-    SUBDIRS += compositor
-    SUBDIRS += client
-    SUBDIRS += cmake
-}
+SUBDIRS += compositor
+SUBDIRS += client
+SUBDIRS += cmake
